irispy/spectrograph.py

```diff
@@ -611,10 +623,18 @@ def read_iris_spectrograph_level2_fits(filenames, spectral_windows=None):
                                     DN_unit, single_file_meta, window_extra_coords,
                                     mask=data_mask))
         hdulist.close()
-    # Construct dictionary of IRISSpectrogramCubeSequences for spectral windows
-    data = dict([(window_name, IRISSpectrogramCubeSequence(data_dict[window_name],
-                                                           window_metas[window_name],
-                                                           common_axis=0))
+    # Construct dictionary of IRISSpectrogramCubeSequences for spectral windows.
+    # The one exception is when a single sit-and-stare file is being read in.
+    # In that case create a dictionary of IRISSpectrogramCubes
+    if top_meta["NRASTERP"] == len(filenames) == 1:
```

Comment:
Is it really best to treat these two cases differently? To me it makes the most sense to leave the sit-and-stare observations in an `IRISSpectrogramCubeSequences` but with only one element. That way, both cases can be treated the same by the user.

Reply:
Good point.  I can remove this change.

Reply:
I have reverted the commit that made this change.  So the WCS fix will still work but the output type will be the same whether the file is sit-and-stare or raster.